cmd/cmd.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/spf13/cobra"
 
 	_ "github.com/elastic/elastic-agent-libs/logp/configure"
-	"github.com/elastic/elastic-agent-shipper/server"
+	"github.com/elastic/elastic-agent-shipper/controller"
```

Comment:
I moved controller-related stuff to a separate package to resolve the dependency loop with `./config`.